README.md

```diff
@@ -124,7 +124,7 @@ See also the list of [contributors](https://github.com/idealista/solrcloud-role/
 
 ## License
 
-![Apache 2.0 Licence](https://img.shields.io/hexpm/l/plug.svg)
```

Comment:
the good licence

Reply:
I'll miss it when we finally remove it from all our repos